backend/src/main/resources/db/migration/V2__create_audit_tables.sql

```diff
@@ -0,0 +1,127 @@
+                                         hq_house_number text,
+                                         hq_postal_code text,
+                                         hq_region text,
+                                         hq_region_code text,
+                                         hq_street text,
+                                         ico text,
+                                         institution_type text,
+                                         lat double precision,
+                                         lawyer_form_code text,
+                                         layer_form text,
+                                         lng double precision,
+                                         person_type text,
+                                         person_type_code text,
+                                         phone_number text,
+                                         postal_code text,
+                                         region text,
+                                         region_code text,
+                                         specialization text,
+                                         street text,
+                                         substitute text,
+                                         title text,
+                                         website text
+);
+
+CREATE TABLE healthcare_provider_category_aud (
+                                                  rev integer NOT NULL,
+                                                  institution_id bigint NOT NULL,
+                                                  location_id bigint NOT NULL,
+                                                  id bigint NOT NULL,
+                                                  revtype smallint
+);
+
+CREATE TABLE selfexamination_record_aud (
+                                            id bigint NOT NULL,
+                                            rev integer NOT NULL,
+                                            revtype smallint,
+                                            due_date date,
+                                            result text,
+                                            status text,
+                                            type text,
+                                            uuid text,
+                                            waiting_to date,
+                                            account_id bigint
+);
+
+ALTER TABLE ONLY account_aud
+    ADD CONSTRAINT pkey_account_aud PRIMARY KEY (id, rev);
+
+ALTER TABLE ONLY badge_aud
+    ADD CONSTRAINT pkey_badge_aud PRIMARY KEY (account_id, type, rev);
+
+ALTER TABLE ONLY examination_record_aud
+    ADD CONSTRAINT pkey_examination_record_aud PRIMARY KEY (id, rev);
+
+ALTER TABLE ONLY healthcare_category_aud
+    ADD CONSTRAINT pkey_healthcare_category_aud PRIMARY KEY (id, rev);
+
+ALTER TABLE ONLY healthcare_provider_aud
+    ADD CONSTRAINT healthcare_provider_aud_pkey PRIMARY KEY (institution_id, location_id, rev);
```

Comment:
Everything is pkey_xxx, just this is xxx_pkey :)

Reply:
Oops, [cc3b501](https://github.com/cesko-digital/loono-be/pull/73/commits/cc3b501e6836d543472f25cb87d7f6e9dfcd7b86)